### PR TITLE
Planned subsystems (v2)

### PR DESCRIPTION
**Pull request by duane701**
All new commits fixuped and lovely.

replaces #4

### CODE REVIEW

File: src/Commands/DefaultShooter.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	void Initialize();
+	void Execute();
+	bool IsFinished();
+	void End();
+	void Interrupted();
+private:
+
+};
+
+#endif  // DefaultShooter_H
```
Comment (OliviaRS) [inline src/Commands/DefaultShooter.h:18]: Why are there comments at the ends of the header files like this?

File: src/Commands/DefaultShooter.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  Requires(Shooter::getInstance().get());
+}
+
+void DefaultShooter::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void DefaultShooter::Execute() {
+	shooter->SetShooter(shooter->GetShooterspeed());
```
Comment (OliviaRS) [inline src/Commands/DefaultShooter.cpp:19]: This line is buggy. Please fix.

File: src/RobotMap.h [hunk 3]
```diff
@@ -6,13 +6,26 @@
...
+  static const int kID1 = 1;
+  static const int kID2 = 2;
+  static const int kID3 = 3;
+  static const int kID4 = 4;
+  static const int kID5 = 5;
+  static const int kID6 = 6;
+  static const int kID7 = 7;
+  static const int kID8 = 8;
+  static const int kID9 = 9;
+  static const int kID16 = 16;
```
Comment (OliviaRS) [inline src/RobotMap.h:27]: In the future, please use more appropriate placeholders.

**Review by OliviaRS — CHANGES_REQUESTED** [review]

File: src/Commands/DefaultShooter.h (hunk 1 above)
Comment (duane701) [reply src/Commands/DefaultShooter.h:18]: This is common with header files. This helps to match up the `#endif` with the `#if`.

**Comment by duane701** [comment]
Closed in favor of #6

**Pull request closed without merging**